### PR TITLE
adding gcp disk kms ci jobs

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @jianli-wei and @liangxia for review

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml [hunk 1]
```diff
@@ -1240,6 +1240,27 @@ tests:
...
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-disconnected-workers-rhel8
+- as: gcp-ipi-disk-encryption-p2-f14
+  cron: 35 0 13,28 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      E2E_RUN_TAGS: '@gcp-ipi and @network-openshiftsdn and not @fips'
+      TAG_VERSION: '@4.10'
+      TEST_ADDITIONAL: Network_Observability
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml:1250]: `TEST_ADDITIONAL` is expected ?

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml]: Thanks for the catch, fixed.

**Pull request merged**